res/rules-engine/configuration.toml

```diff
@@ -63,3 +63,20 @@ Type = "zero"
   Port = 5566
   Protocol = "tcp"
 
```

Comment:
need to remove `ConfigAccessTokenFile ` and `AccessTokenFile ` from the above config